src/SuperHero.java

```diff
@@ -1,4 +1,52 @@
 public class SuperHero {
+    int healthPoints = 100;
+    int attackPower;
+    boolean isTurningIntoStatue;
```

Comment:
Turning is a process, here better option can be

`isTurnedIntoStatue`
`isAStatue`